plugin/src/sbt-test/sbtix/simple/test

```diff
@@ -1,5 +1,10 @@
 > compile
 > genNix
-$ exists repo.nix
+> reload plugins
+> genNix
+$ exec mv repo.nix repo-build.nix
```

Comment:
I was considering setting something up to merge all these files together and drop the additional commands too.  So there'd only be `sbtix-gen` and `repo.nix`.  

So I think we could have a script that recurses into the project directory and counts how deep it has to go until it doesn't see any sbt/scala files. Then it would know how many times to run `genNix`.  `sbtix-gen` could do that. And the files could all be left in their own project directories.  I'm not sure there's any good reason to keep `sbtix-gen` and `sbtix-gen-all` as separate commands.

We could then have `sbtix-merge` and/or `sbtix-gen-merge`. One merges the files into a repo.nix. The other generates and merges in one go.

I find myself deleting repo.nix a lot. We should probably name it something we can add to .gitignore.  We could call it `repo-sbtix-gen.nix` to match the name of the command or `repo-sbtix.nix`  I was avoiding having `manual-repo.nix` and `sbtix.nix` have names with the same first few characters as the generated `repo.nix` files since it would increase the risk of deleting the wrong ones.